ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_SERVERS.yml

```diff
@@ -389,6 +389,71 @@ servers:
           mode: active
           enabled: false
 
+  server18_monitoring_session_source:
+    rack: RackC
+    adapters:
+      - endpoint_ports: [ Eth1 ]
+        switch_ports: [ Ethernet25, Ethernet26 ]
+        switches: [ DC1-SVC3A ]
```

Comment:
This data set is not valid. The length of `switch_ports`, `endpoint_ports` and `switches` should be the same.
This is the reason why your templates doesn't create unexpected data.
When you add two switches with one interface each, you will see my point with one or multiple interfaces. the current templates will output the monitor session for both interfaces on both devices.